packages/account/src/Components/poa/common-mistakes/common-mistake-examples.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { getExampleImagesConfig } from '../../../Configs/poa-common-mistake-examples-config';
+import IcErrorBadge from '../../../Assets/ic-error-badge.svg';
+import './common-mistake-examples.scss';
+
+type TCommonMistakeExamplePartialsProps = {
+    description: string;
+    image: React.ReactElement;
+};
+
+/**
+ * Wrapper for displaying the image and description of the mistake
+ * @name CommonMistakeExamplePartials
+ * @param description - description of the mistake
+ * @param image - Sample image
+ * @returns  {React.ReactElement}
+ */
+const CommonMistakeExamplePartials = ({ description, image }: TCommonMistakeExamplePartialsProps) => (
+    <div className='common-mistake-examples__content-layout'>
+        {image}
+        <div className='common-mistake-examples__content-description'>
+            <IcErrorBadge />
+            <Text size='xxxs' line_height='s' role='document'>
+                {description}
+            </Text>
+        </div>
+    </div>
+);
+
+/**
+ * Displays the image and description of the mistake
+ * @returns {React.ReactElement}
+ */
+const CommonMistakeExamples = () => {
+    const example_images = getExampleImagesConfig();
+    return (
+        <React.Fragment>
+            <Text as='div' weight='bold' size={isMobile() ? 'xxs' : 'xs'} className='common-mistake-examples__title'>
+                {localize('Common mistakes')}
+            </Text>
+            <div className='common-mistake-examples__content'>
+                {example_images.map(config => (
```

Comment:
```suggestion
                {example_images.map(({ ref, description, image }) => (
```

Reply:
thanks for the suggestion. I would prefer to not extract variables. As I see, mostly we don’t do it in our codebase. can we stay with it, if you don’t mind? :)

Reply:
It's more cleaner to use destructuring but anyway it's your choice :smile: 

https://www.turing.com/kb/writing-clean-react-code#:~:text=by%20the%20browser.-,Use%20destructuring,-Destructuring%20props%20and